packages/shared-business/src/components/AccountStatement.tsx

```diff
@@ -183,15 +210,15 @@ export const AccountStatementV1 = ({
           <Space height={48} />
           <Box direction="row" justifyContent="spaceBetween">
             <Box direction="column">
-              <Text style={styles.sectionTitle}>
-                {t("accountStatement.date", { openingDate, closingDate })}
+              <Text style={styles.dateTitle}>
+              {t("accountStatement.date", { openingDate, closingDate })}
               </Text>
             </Box>
 
             <Box direction="column">
               <Text style={styles.openingBalanceText}>{t("accountStatement.openingBalance")}</Text>
               <Text style={styles.totalAmount}>
-                {formatCurrencyIso(Number(openingBalance.value), openingBalance.currency)}
+                {openingBalance.value}
```

Comment:
is it formatted with currency on backend side now ? Or we don't want to display the currency anymore there ?